config/environments/development.rb

```diff
@@ -19,6 +19,9 @@
   config.action_mailer.delivery_method = :smtp
   config.action_mailer.raise_delivery_errors = true
   config.action_mailer.default_url_options = { host: 'localhost:3001' }
-  config.action_mailer.smtp_settings = { address: ENV['SMTP_HOST'] || 'localhost', port: ENV['SMTP_PORT'] || 2025 }
+  config.action_mailer.smtp_settings = {
+    address: ENV['SMTP_HOST'] || 'localhost',
+    port: ENV['SMTP_PORT'] || 2025
```

Comment:
I'd love to do that! But perhaps in a separate PR so we can keep the separate threads separate (1 PR for updating styles for current rubocop config, one for updating rubocop config rules)